### PR TITLE
*:change the reserve space description

### PR DESCRIPTION
**Pull request by tier-cap**
Signed-off-by: tier-cap <zhengxiaojin@pingcap.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
- Modified parameter meaning and usage description 

This PR has updated the description about `reserve-space` from https://github.com/pingcap/docs-cn/pull/6845 that was reverted.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/tikv/tikv/pull/10833/checks

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -331,7 +331,9 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 
 ### `reserve-space`
 
-+ TiKV 启动时预占额外空间的临时文件大小。临时文件名为 `space_placeholder_file`，位于 `storage.data-dir` 目录下。TiKV 磁盘空间耗尽无法正常启动需要紧急干预时，可以删除该文件，并且将 `reserve-space` 设置为 `0MB`。
++ TiKV 启动时会预留一块空间用作磁盘空间保护，当磁盘剩余空间不足该值时，TiKV 会对业务写操作进行限制。预留空间形式上包括两个部分，`80%` 用作空间不足时清理回收空间等运维操作所需要的额外磁盘空间，剩余 `20%` 为实际的物理文件，用于回收空间过程中额外空间占用过多而耗尽存储的最后一道防御。
++ 该物理文件名为 `space_placeholder_file`，位于 `storage.data-dir` 目录下。TiKV 磁盘空间耗尽无法正常启动需要紧急干预时，可以删除该文件，并且将 `reserve-space` 设置为 `0MB`。
++ 防御的有效性与取值大小有关联，实际预留大小为磁盘容量的 `5%` 与该配置项之间的最大值。
```
Comment (en-jin19) [inline tikv-configuration-file.md:336]:
```suggestion
+ TiKV 启动时会预留一块空间用于保护磁盘空间。当磁盘剩余空间小于该预留空间时，TiKV 会对业务写操作进行限制。预留空间形式上分为两个部分：80% 的预留空间用作磁盘空间不足时运维操作（如清理和回收空间）所需要的额外磁盘空间，剩余的 20% 的预留空间用于保存实际的物理文件。在回收空间的过程中，如果额外磁盘空间占用了过多空间，导致存储耗尽时，存放物理文件的 20% 的预留空间会成为最后一道防御。
+ 该物理文件名为 `space_placeholder_file`，位于 `storage.data-dir` 目录下。当 TiKV 磁盘空间耗尽无法正常启动需要紧急干预时，可以删除该文件，并且将 `reserve-space` 设置为 `0MB`。
+ 防御的有效性与预留空间的大小相关。预留空间的实际大小为磁盘容量的 5% 与该配置项的值之间的最大值。
```

Comment (tier-cap) [reply tikv-configuration-file.md:336]: + TiKV 启动时会预留一块空间用于保护磁盘空间。当磁盘剩余空间小于该预留空间时，TiKV 会对业务部分写操作进行限制。预留空间形式上分为两个部分：预留空间的 80% 用作磁盘空间不足时的运维操作所需要的额外磁盘空间，剩余的 20% 为磁盘临时文件。在回收空间的过程中，如果额外使用的磁盘空间过多，导致存储耗尽时，该临时文件会成为恢复服务的最后一道防御。
+ 临时文件名为 `space_placeholder_file`，位于 `storage.data-dir` 目录下。当 TiKV 因磁盘空间耗尽而下线时，重启 TiKV 会自动删除该临时文件，并自动尝试回收空间。
+ 当剩余空间不足时，TiKV 不会再创建该临时文件。
+ 防御的有效性与预留空间的大小有关。预留空间大小的计算方式为磁盘容量的 5% 与该配置项的值之间的最大值。

Comment (tier-cap) [reply tikv-configuration-file.md:336]: > 会限制业务部分进行写操作。
改为：会限制部分写操作。
335 与 336 行建议合并，并去除原 336 行中的 "再"
原 337 行 增加一句： 当配置为 `0MB` 时磁盘防护功能关闭。
y

**Review by NingLin-P — APPROVED** [review]
LGTM

**Review by en-jin19 — APPROVED** [review]
LGTM

**Pull request merged**